testutil/integration/simnet_test.go

```diff
@@ -155,25 +164,30 @@ func TestSimnetDuties(t *testing.T) {
 				args.BMockOpts = append(args.BMockOpts, beaconmock.WithDeterministicSyncCommDuties(2, 2))
 			}
 
+			if test.pregenerateRegistration {
```

Comment:
so `test.pregenerateRegistration` basically disables vmock, rather do that directly. Suggest configuring specific VC type per test and in this case, no VC type at all.

Reply:
Done!